### ISSUE TITLE
Android M

### ISSUE THREAD

**Issue by Darklightning167** [SecrecySupportTeam/secrecy#143]
secrecy stuck at adding to vault on Android M

**Comment by Darklightning167** [comment]
I managed to create a debug log via android studio 
http://pastebin.com/y0NXsWtc

**Comment by L-Henke** [comment]
Line 57-58 state that protobuf cannot deserialize the vault header. Either your vault header is broken and this can be 'fixed' by creating a new vault, or the storage layer in Android M has changed again and Secrecy just can't access that file. The storage layer will be rebuild, but I honestly don't think that will happen too soon.

**Comment by Darklightning167** [comment]
This happened when creating a vault so it's probably the storage layer changes in Android M

**Comment by L-Henke** [comment]
I guess in that case we can only start to adapt to these changes when the API is stable and documented.

**Comment by L-Henke** [comment]
Have you used secrecy before updating to Android M? Otherwise it would be interesting if the SECRECYFILES folder is created, because nothing in the log files shows signs of an error. Which could mean that the folder is already created, or an error when creating that folder is not logged.

**Comment by Darklightning167** [comment]
Yes but it was a clean install of Android M but the SECRECYFILES folder is still created after opening the app for the first time

**Comment by Darklightning167** [comment]
Hey guys just a heads up secrecy seems to be working normally again I'm preview 3
